DangProject/Sources/Data Layer/Repositories/CoreDataManager/DefaultCoreDataManagerRepository.swift

```diff
@@ -116,4 +119,70 @@ class DefaultCoreDataManagerRepository: CoreDataManagerRepository {
             return false
         }
     }
+    
+    func createAlarmEntity(_ alarm: AlarmDomainModel) {
+        guard let context = self.context,
+              let entity = NSEntityDescription.entity(forEntityName: CoreDataName.alarm.rawValue, in: context),
+              let alarmEntity = NSManagedObject(entity: entity, insertInto: context) as? Alarm else { return }
+        alarmEntity.isOn = alarm.isOn
+        alarmEntity.title = alarm.title
+        alarmEntity.message = alarm.message
+        alarmEntity.time = alarm.time
+        alarmEntity.identifier = alarm.identifier
+        alarmEntity.selectedDays = alarm.selectedDaysOfTheWeek
+        
+        do {
+            try context.save ()
+        } catch {
+            print(error.localizedDescription)
+        }
+    }
+    
+    func readTotalAlarmEntity() -> [AlarmDomainModel] {
+        let result = loadFromCoreData(request: Alarm.fetchRequest())
+        let alarmDomainModelArray = result.map{ AlarmDomainModel.init(alarmEntity: $0) }
+        return alarmDomainModelArray.sorted { $0.time < $1.time }
+    }
+    
+    func updateAlarmEntity(_ alarm: AlarmDomainModel) {
+        let request = Alarm.fetchRequest()
+        request.predicate = NSPredicate(format: "identifier == %@", alarm.identifier)
+        do {
+            if let checkedAlarmEntity = try context?.fetch(request) {
+                if checkedAlarmEntity.count == 0 {
+                    return
```

Comment:
```suggestion
            if let checkedAlarmEntity = try context?.fetch(request),
            checkedAlarmEntity.count != 0 {
```

checkedAlarmEntity.count가 0일 땐 아무동작도안하고 return 하는 것 같아보여요
checkedAlarmEntity가 비어있지 않을 때만 동작하면 되는 코드라면
위 if문과 ,로 엮어서 한번에 분기하는 게 어떨까요

Reply:
더 좋은방법이네요 반영했습니다~
77dc8d7cdd0c261f67514e3376e64cf32ee718e4